### ISSUE TITLE
Runtime crash when printing relative pointers

### ISSUE THREAD

**Issue by JoshuaManton** [odin-lang/Odin#671]
## Context

* Operating System: Windows 10
* Odin version/Commit:

## Expected Behavior

I expect relative pointers to print in some nice way

## Current Behavior

My program crashes if I call print with something containing a relative pointer

### Steps to Reproduce

```
package test

import "core:fmt"

Base :: struct {
    position: [3]f32,
    derived: union {
        Derived,
    },
}

Derived :: struct {
    base: #relative(i16) ^Base,
    name: string,
}

main :: proc() {
    thing: Base;
    thing.derived = Derived{nil, "John"};
    thing.derived.(Derived).base = &thing;
    fmt.println(thing); // comment this out to stop the crash from occurring
}
```

compiles fine but crashes at runtime without printing

**Comment by JoshuaManton** [comment]
Just tried it again on the latest version of Odin (94ba1826913893d11d3c373ea0136f773c6218ae) and it is still happening. Note that I am _not_ using -llvm-api. With the `fmt.println(thing);` in there the program crashes at runtime but if I comment it out then I get:
```
d:\programming\odin\src\ir.cpp(6236): Panic: NOT FOUND ir_type_info_index u64 @ index 103
```

**Comment by gingerBill** [comment]
Fixed in c783840
